modules/ROOT/pages/spring/spring-module.adoc

```diff
@@ -5,6 +5,20 @@ endif::[]
 
 The Spring module allows Mule apps to make use of the Spring Framework.
 
+== To Install the Spring module
```

Comment:
Set in title case and remove "To":  == Install the Spring Module

Reply:
Fixed in d1b2c1e